src/events.cr

```diff
@@ -138,7 +138,8 @@ module Office365::Events
     calendar_id : String? = nil,
     period_start : Time = Time.local.at_beginning_of_day,
     period_end : Time? = nil,
-    ical_uid : String? = nil
+    ical_uid : String? = nil,
+    top : Int? = 10000
```

Comment:
```suggestion
    top : Int32? = 10000
```

Reply:
Int is an abstract class, to write integer without a trace like _i64 or _u64, we need to declare it as `Int32`